docker/Makefile

```diff
@@ -49,53 +49,53 @@ endif
 
 deploy-dev:
 	# generate self-signed certificates
-	docker run --rm -t -v "$(BASE_DIR):/athenz" --user "$(shell id -u):$(shell id -g)" athenz-setup-env sh /athenz/docker/setup-scripts/self-signed-certificates.sh
+	docker run --rm -t -v "$(BASE_DIR):/athenz" --user "$(shell id -u):$(shell id -g)" athenz-setup-env bash /athenz/docker/setup-scripts/self-signed-certificates.sh
```

Comment:
I changed from "sh" to "bash" because executing the script with "sh" caused the following error.

```
make deploy-local                                                                                                 master ✭
# generate self-signed certificates
docker run --rm -t -v "/Users/<user>/ghq/ghe.corp.yahoo.co.jp/athenz/athenz:/athenz" --user "504:20" athenz/athenz-setup-env sh /athenz/docker/setup-scripts/self-signed-certificates.sh
/athenz/docker/setup-scripts/self-signed-certificates.sh: 4: set: Illegal option -o pipefail
Error: exit status 2
make: *** [Makefile:64: deploy-local] Error 1
```